<?xml version="1.0"?>

<launch>

<node pkg="tf" type="static_transform_publisher" name="ultrasonic1" args="1 1 0 0 0 0 1 base_link ultrasonic1 100" />
<node pkg="tf" type="static_transform_publisher" name="ultrasonic2" args="1 2 0 0 0 0 1 base_link ultrasonic2 100" />
<node pkg="tf" type="static_transform_publisher" name="ultrasonic3" args="1 3 0 0 0 0 1 base_link ultrasonic3 100" />
<node pkg="tf" type="static_transform_publisher" name="ultrasonic4" args="1 4 0 0 0 0 1 base_link ultrasonic4 100" />

<node pkg="rviz" type="rviz" name="rviz" args="-d $(find ultrasonics)/rviz/ultrasonics.rviz"/>
<node pkg="ultrasonics" type="manueldriver.py" name="ultrasonics" output="screen"> </node>

</launch>
